src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -50,6 +51,36 @@ public sealed class Arguments : TransformInputBase
             public string[] OutputColumns;
         }
 
+        public sealed class TrainingArguments
```

Comment:
Did you forgot to add Arguments as base class, or it's intended?

Reply:
I was thinking to keep it separate from `Arguments` class initially but I think it make sense to derive it from `Arguments` class.

---
In reply to: [221764623](https://github.com/dotnet/machinelearning/pull/1063#discussion_r221764623) [](ancestors = 221764623)